mithril-client/src/commands/snapshot/show.rs

```diff
@@ -27,39 +30,51 @@ impl SnapshotShowCommand {
         let config: Config = config_builder.build()?;
         let mut dependencies_builder = DependenciesBuilder::new(Arc::new(config));
         let snapshot_service = dependencies_builder.get_snapshot_service().await?;
-        let snapshot = snapshot_service.show(&self.digest).await?;
+        let signed_entity = snapshot_service.show(&self.digest).await?;
 
         if self.json {
-            println!("{}", serde_json::to_string(&snapshot)?);
+            println!("{}", serde_json::to_string(&signed_entity.artifact)?);
```

Comment:
Loss of the two removed fields in the json output, maybe use a SnapshotMessage instead ? (This means using a adapter)